perception_config:
    trigger_roi: [[125, 335], [160, 375]]
    min_blk_px_cnt: 200 # minimum num of pixel required to be black for the tile to be non empty
    adaptive_threshold_block_size: 75
    adaptive_threshold_C: 10
    aruco_marker_ideal_ids: [11, 3, 7, 10] # order of aruco marker ids (from bottom left of image in counter clockwise order)
